hermes-schema/src/main/java/pl/allegro/tech/hermes/schema/SchemaRepository.java

```diff
@@ -39,7 +39,17 @@ public SchemaRepository(SchemaVersionsRepository schemaVersionsRepository,
      * from underlying schema repository.
      */
     public CompiledSchema<Schema> getKnownAvroSchemaVersion(Topic topic, SchemaVersion version) {
-        return compiledAvroSchemaRepository.getSchema(topic, version);
+        return getCompiledSchemaAtVersion(topic, version);
+    }
+
+    private CompiledSchema<Schema> getCompiledSchemaAtVersion(Topic topic, SchemaVersion latestVersion) {
+        try {
+            return compiledAvroSchemaRepository.getSchema(topic, latestVersion);
+        } catch (CouldNotLoadSchemaException e) {
```

Comment:
\[PMD\] INFO: A catch statement that catches an exception only to rethrow it should be avoided\.